src/scenes/OrgRouter/NavBar.tsx

```diff
@@ -196,20 +197,37 @@ export default function NavBar({
     [closeAndNavigateTo, isDeliverablesRoute, isDeliverableViewRoute, hasDeliverables]
   );
 
+  // TODO: fetch reports from API and replace reference to seedFundReports in reportsMenu
+  const reportsMenu = useMemo<JSX.Element | null>(
+    () =>
+      seedFundReports.length > 0 && selectedOrganization.canSubmitReports ? (
+        <NavItem
+          icon='iconGraphReport'
+          label={strings.REPORTS}
+          selected={!!isReportsRoute}
+          onClick={() => {
+            closeAndNavigateTo(APP_PATHS.REPORTS);
+          }}
+          id='reports-list'
+        />
+      ) : null,
+    [closeAndNavigateTo, isReportsRoute, seedFundReports.length, selectedOrganization.canSubmitReports]
+  );
```

Comment:
I'm not very familiar with `selectedOrganization.canSubmitReports`. Would we still want to check this value for non seed fund reports? Or would we potentially need a new value/flag for this report type?

Reply:
We used to enable the possibility of submitting reports to orgs, but maybe now it will apply to all orgs? @alexTerraformation may know that answer better

Reply:
I think we need to check the feature flag here

Reply:
Oops! I completely forgot to implement the feature flag in this PR. Adding now, thanks @constanzauanini 🙌